enterprise-ws-spec/src/main/asciidoc/ClientProgrammingModel.adoc

```diff
@@ -369,60 +349,28 @@ interface argument to a port is not declared in the client deployment
 descriptor, the container may provide a default resolution capability or
 throw a ServiceException.
 
-===== Dynamic Port access
-
-With JAX-RPC, a client may use the following DII methods of a Service
-Interface located by a JNDI lookup of the client’s environment to obtain
-a Call object:
-
-Call createCall() throws ServiceException;
-
-Call createCall(QName portName) throws ServiceException;
-
-Call createCall(QName portName, String operationName) throws
-ServiceException;
-
-Call createCall(QName portName, QName operationName) throws
-ServiceException;
-
-Call[] getCalls(QName portName) throws ServiceException;
-
-A DII Call object may or may not be pre-configured for use depending on
-the method used to obtain it. See the JAX-RPC specification for details.
-
-These methods are not supported in JAX-WS. JAX-WS provides Dispatch APIs
-to enable the client to operate at XML message level. See section 4.2.6.
-
 ===== ServiceFactory
 
-Use of the JAX-RPC ServiceFactory class is not recommended in a Web
-Services for Java EE product. A Web Services for Java EE client must
-obtain a Service Interface/Class using JNDI lookup as described in
-section link:#anchor-26[4.2.1]. Container providers are not required to
-support managed Service instances created from a ServiceFactory.
-
-ServiceFactory class has been removed from JAX-WS. It has been replaced
-by two static methods Service.create(QName serviceName)and
+ServiceFactory class has been removed from Jakarta XML Web Services. It has
```

Comment:
I do not think ServiceFactory has ever been part of JAX-WS, in any case it is not part of the last available nor current version. What about:
* changing section title to "Service Factory", so it refers more to design pattern then a class name
* reword the section to start with sth like:
"Two static factory methods Service.create(QName serviceName) and Service.create(URL wsdlLocation, QName serviceName) for creating Service instances  rely on specific implementations of ServiceDelegate Class in any Jakarta XML Web Services compliant implementation. The use of these static methods is not recommended..."